system-int-test/test/cmr/system_int_test/ingest/variable_ingest_test.clj

```diff
@@ -12,28 +14,25 @@
 (use-fixtures :each (ingest-util/reset-fixture))
 
 (deftest create-variable-ingest-test
-  (testing "ingest a new variable"
-    (let [;; Groups
-          update-group-id (e/get-or-create-group (s/context) "umm-var-guid1")
-          ;; Tokens
-          update-token (e/login (s/context) "umm-var-user1" [update-group-id])
-          ;; Grants
-          update-grant-id (e/grant-group-admin
-                           (assoc (s/context) :token update-token)
-                           update-group-id
-                           :update)
-          variable-data (variable-util/make-variable)]
-      (is (e/permitted? update-token
-                        update-grant-id
-                        update-group-id
-                        (ingest-util/get-ingest-update-acls update-token)))
-      (let [{:keys [status concept-id revision-id]} (variable-util/create-variable
-                                                     update-token variable-data)]
+  (testing "ingest:"
+    (let [acl-data (variable-util/setup-update-acl (s/context))
+          {:keys [user-name group-name group-id token grant-id]} acl-data
+          variable-data (variable-util/make-variable)
+          response (variable-util/create-variable token variable-data)
+          {:keys [status concept-id revision-id]} response
+          fetched (mdb/get-concept concept-id revision-id)
+          metadata (edn/read-string (:metadata fetched))]
+      (testing "create a new variable"
+        ;; Sanity check on tokens; we've had issues with ACLs, so the following is
+        ;; handy when firest testing ingest. More complete ACL testing is done
+        ;; below as well as in other sys-int test namespaces (e.g., associations).
+        (is (e/permitted? token
+                          grant-id
+                          group-id
+                          (ingest-util/get-ingest-update-acls token)))
         (is (= 201 status))
         (is (= 1 revision-id))
         (is (mdb/concept-exists-in-mdb? concept-id revision-id))
-        (is (= (:name variable-data)
-               (:name (mdb/get-concept concept-id revision-id))))
         ;; XXX Once CMR-4172 (metdata-db services support) was added, we tried to
```

Comment:
Okay, null exception is gone now. This test is working and it fails in the same way that @chris-durbin objected to in the delete-variable branch. Once we sort out what's up with the `:created-at` and `:extra-fields` for delete, this will be passing.